Blockzilla/Modules/WebView/WebViewController.swift

```diff
@@ -253,6 +253,12 @@ class WebViewController: UIViewController, WebController {
         browserView.evaluateJavaScript(javascript, completionHandler: completion)
     }
 
+    func evaluatePDFContentType(_ onCompletion: @escaping (Bool) -> Void) {
+        evaluate("document.contentType") { result, _ in
+            onCompletion(result is String && (result as? String) == "application/pdf")
+        }
```

Comment:
Checking if the text is String and then casting is the same thing as only casting it.